security/policies-and-procedures.md

```diff
@@ -0,0 +1,12 @@
+# Policies and Procedures
+
+Currently the team is working to finalize this work and have it completed for the 1.8 Release and for the transition to the CNCF.
+These policies and procedures will be based of the [Kubernetes Policy](https://kubernetes.io/docs/reference/issues-security/security/)
+We are working on updating and editing these policies and procedures for the Kubeflow Community.  Our very **rough DRAFT and working docs** can be found [here](https://docs.google.com/document/d/1sjWa0G2UqdsP1QROYVEl9iQR6v1x8HD7-F4TQo7lV6M/edit?usp=sharing)
+
+Kubeflow Security Team [Policy and Procedure Working Document] (https://docs.google.com/document/d/1vw_efQyYG_zWEoL-vk9mZQX5p7fOcl3RTG6pcxMhYKI/edit?usp=sharing) - Please note this is only a **WORKING** Document. 
+
+We are always looking for Kubeflow community users and contributoes to help us. If you would like to help the security team with these docs or other items please consider doing the following:
```

Comment:
Typo: contributors

Reply:
@james-jwu - Thank you! Fixed this typo.